### ISSUE TITLE
[Google Trends API] Response has slight different value from Goole Trends

### ISSUE THREAD

**Issue by hilmanski** [serpapi/public-roadmap#1753]
After a few tests, I noticed that we returned slightly different values from the actual data from Google Trends.

From SerpApi: [Inspect](https://serpapi.com/searches/668b7a368b7ed29454fc475d/inspect)
![CleanShot 2024-07-08 at 13 37 34@2x](https://github.com/serpapi/public-roadmap/assets/4522467/92fdb466-4e95-4fc8-83c5-e5136f6d79b4)

VS value from Google Trends.

![CleanShot 2024-07-08 at 13 38 38](https://github.com/serpapi/public-roadmap/assets/4522467/67c0f8a5-91d8-43d2-b877-b5bf33bf5f21)

*This data comes from this endpoint `https://trends.google.com/trends/api/widgetdata/multiline..` upon accessing the same page on Google Trends. The numbers match what's on the visual.

**Comment by ilyazub** [comment]
It looks like a duplicate of https://github.com/serpapi/public-roadmap/issues/887.

**Comment by schaferyan** [comment]
Correct me if I'm wrong, but I don't think it's an exact duplicate. https://github.com/serpapi/public-roadmap/issues/887 was initially about random 0 values, but it morphed into a catch-all for variation from the expected response. I suspect there are multiple issues causing Google Trends to return different values, and it may be helpful to try to document them separately.

**Comment by hilmanski** [comment]
I agree with @schaferyan , while both address the incorrect information. The #887 focusing on the random 0 values. While this one addresses the wrong points, I'm afraid it can be overlooked since the numbers are very close.

**Comment by marm123** [comment]
Another user reported this:

[Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/102910000070611?view=List)

**Comment by hilmanski** [comment]
A user reported this issue.

[Intercom](https://serpapi.com/searches/67078265e45f33494c75cfc3/inspect)

**Comment by tanys123** [comment]
@hilmanski Yea this has been an issue. I can't confirm if this is a usual Google behaviour. Usually they only off by a small margin, so the trend line still looks identical. I wonder if matching exact value is more important or the overall trends.

> ... \[Intercom](https://serpapi.com/searches/67078265e45f33494c75cfc3/inspect)

Inspect link is added instead, can you add the Intercom link?

**Comment by sonika-serpapi** [comment]
The same user Hilman mentioned above, also mentioned that the slight difference (1 point) could be attributed to the time zone factor.

**Comment by sonika-serpapi** [comment]
Another user reported this:

[Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/102910000071094?view=List)

**Comment by marm123** [comment]
Another customer reported this:

[Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/215468895156892)

**Comment by oceansize** [comment]
Another user reported this:

[Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/215468992385841?view=List)

**Comment by oceansize** [comment]
Another customer reported this:

[Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/215469166267103?view=List)

**Comment by dperlman** [comment]
Please see [my comment on issue #887](https://github.com/serpapi/public-roadmap/issues/887#issuecomment-3133822388) 

TLDR Google Trends adds random noise to the data. So you shouldn't expect multiple requests to return exactly the same thing.

The random noise may be an artifact of their sampling process, or it may be intentional. Only Google knows. But it's there and we have to work with it. And it's definitely not a bug in SerpApi.
